webdriver/tests/bidi/network/add_intercept/add_intercept.py

```diff
@@ -58,6 +58,58 @@ async def test_other_context(
     await fetch(text_url, context=other_context)
 
 
+@pytest.mark.asyncio
+async def test_other_context_with_event_subscription(
```

Comment:
Some other test suggestions:
- add an intercept for 2 contexts and check requests are intercepted for both contexts
- add an intercept for 1 context and another global intercept, check that a request for the context has both intercept ids in the intercepts array

Reply:
@OrKoN just a quick ping for those additional test suggestions. Can be done in a followup as well, so I'll approve the PR for now.

Reply:
I will open a separate PR for that since touching modules causes too many noisy failures and makes it difficult to see only new test results